lib/urlToRequest.js

```diff
@@ -30,6 +30,9 @@ function urlToRequest(url, root) {
 			default:
 				throw new Error("Unexpected parameters to loader-utils 'urlToRequest': url = " + url + ", root = " + root + ".");
 		}
+	} else if(/^[a-z][a-z0-9+\-.]*:\/\//.test(url)) {
```

Comment:
This RegExp is very liberal. I would like to be more conservative here since there are only three reasonable variations here:

- `http://`
- `https://`
- `//` (protocol agnostic)

I can't think of a use-case where other protocols are useful.

So could you change that RegExp to `/^(https?:)?\/\//i`?